res/controllers/Hercules-DJControl-Inpulse-300-script.js

```diff
@@ -65,25 +65,17 @@ DJCi300.kScratchActionScratch = 1;
 DJCi300.kScratchActionSeek = 2;
 
 // Pad modes
-DJCi300.padModeNone = -1;
-DJCi300.padModeHotcue = 0;
-DJCi300.padModeRoll = 1;
-DJCi300.padModeSlicer = 2;
-DJCi300.padModeSampler = 3;
-DJCi300.padModeToneplay = 4;
-DJCi300.padModeFX = 5;
-DJCi300.padModeSlicerloop = 6;
-DJCi300.padModeBeatjump = 7;
-
-// Slicer connections
-let slicerSizeConnection1;
-let slicerTrackConnection1;
-let slicerBeatConnection1;
-let slicerTrackConnection2;
-let slicerSizeConnection2;
-let slicerBeatConnection2;
-
-DJCi300.vuMeterUpdateMaster = function(value, _group, _control) {
+DJCi300.padModeNone = 0;
+DJCi300.padModeHotcue = 15;
```

Comment:
Add a comment saying that these numbers correspond directly to the midi control values